### PR TITLE
Tests consistency improvements

### PR DESCRIPTION
**Pull request by omaralbeik**

### CODE REVIEW

File: Tests/CoreGraphicsTests/CGColorExtensionsTests.swift [hunk 1]
```diff
@@ -7,29 +7,35 @@
 //
 
 import XCTest
+@testable import SwifterSwift
+
+#if canImport(CoreGraphics)
```
Comment (LucianoPAlmeida) [inline Tests/CoreGraphicsTests/CGColorExtensionsTests.swift]: Awesome @omaralbeik 💯

File: Tests/UIKitTests/UINavigationBarExtensionTests.swift [hunk 2]
```diff
@@ -6,11 +6,12 @@
 //  Copyright © 2017 SwifterSwift
 //
 
-#if os(iOS) || os(tvOS)
-
 import XCTest
 @testable import SwifterSwift
 
+#if canImport(UIKit)
+import UIKit
```
[the remark this answers is not in the document]
Comment (SD10) [reply Tests/UIKitTests/UINavigationBarExtensionTests.swift]: IIRC, we don't do any unit testing for watchOS

File: Tests/CoreGraphicsTests/CGSizeExtensionsTests.swift [hunk 3]
```diff
@@ -27,73 +30,75 @@ final class CGSizeExtensionsTests: XCTestCase {
 		XCTAssertEqual(newRect.height, 60)
 	}
 
-    func testAdd() {
```
[the remark this answers is not in the document]
Comment (omaralbeik) [reply Tests/CoreGraphicsTests/CGSizeExtensionsTests.swift]: https://www.youtube.com/watch?v=SsoOG6ZeyUI

File: Tests/UIKitTests/UINavigationBarExtensionTests.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (omaralbeik) [reply Tests/UIKitTests/UINavigationBarExtensionTests.swift]: I revised all files and added `#if !os(watchOS)` where watchOS is not supported

**Pull request merged**